From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: George T Kramer <george.t.kramer@intel.com>
Date: Thu, 2 Aug 2018 14:43:28 -0700
Subject: [PATCH] Use unified trust store

Expose user-managed CA certs to applications that require the certifi
package by using the unified trust store.  This has a bonus side-effect
of no longer distributing a copy of the Mozilla CA certs and instead
relying on those distributed as a part of the ca-certs package.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 MANIFEST.in                  | 2 +-
 certifi.egg-info/SOURCES.txt | 3 +--
 certifi/core.py              | 4 +---
 setup.py                     | 1 -
 4 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/MANIFEST.in b/MANIFEST.in
index 6077b5f..24ad572 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1 +1 @@
-include MANIFEST.in README.rst LICENSE certifi/cacert.pem
+include MANIFEST.in README.rst LICENSE
diff --git a/certifi.egg-info/SOURCES.txt b/certifi.egg-info/SOURCES.txt
index 04773ea..f5b8172 100644
--- a/certifi.egg-info/SOURCES.txt
+++ b/certifi.egg-info/SOURCES.txt
@@ -5,10 +5,9 @@ setup.cfg
 setup.py
 certifi/__init__.py
 certifi/__main__.py
-certifi/cacert.pem
 certifi/core.py
 certifi.egg-info/PKG-INFO
 certifi.egg-info/SOURCES.txt
 certifi.egg-info/dependency_links.txt
 certifi.egg-info/not-zip-safe
-certifi.egg-info/top_level.txt
\ No newline at end of file
+certifi.egg-info/top_level.txt
diff --git a/certifi/core.py b/certifi/core.py
index 56b52a3..ab43d68 100644
--- a/certifi/core.py
+++ b/certifi/core.py
@@ -21,9 +21,7 @@ except ImportError:
 
 
 def where():
-    f = os.path.dirname(__file__)
-
-    return os.path.join(f, "cacert.pem")
+    return '/var/cache/ca-certs/compat/ca-roots.pem'
 
 
 def contents():
diff --git a/setup.py b/setup.py
index f1cfef4..eff96bf 100755
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,6 @@ setup(
         'certifi',
     ],
     package_dir={'certifi': 'certifi'},
-    package_data={'certifi': ['*.pem']},
     # data_files=[('certifi', ['certifi/cacert.pem'])],
     include_package_data=True,
     zip_safe=False,
